### ISSUE TITLE
prescribedHeatFlowRate in PartialEffectiveness

### ISSUE THREAD

**Issue by Mathadon** [ibpsa/modelica-ibpsa#907]
`IBPSA.Fluid.HeatExchangers.BaseClasses.PartialEffectiveness` computes `Q_flow` from the fluid inlet temperatures, which do not depend on `Q_flow`. Therefore this model should have `prescribedHeatFlowRate1=prescribedHeatFlowRate2=true`. Otherwise the model can produce division of zero by zero. A unit test should be added that produces this behavior and the model should be fixed.

**Comment by Mathadon** [comment]
The following model produces the error:

```
model Test
  extends ConstantEffectiveness(PSin_1(height=0, offset=1E5));
end Test;
```

```
Error: Scalar system is always singular for hex.port_a1.h_outflow = (hex.Q1_flow-60499.614*sin_1.ports[1].m_flow)/((-1.0)*sin_1.ports[1].m_flow) = -0/-0
```

but it seems a bit dull to add this model just for testing how it behaves at zero flow? 

So I'll only change the default values of `prescribedHeatFlowRate1=prescribedHeatFlowRate2=true` in `PartialEffectiveness` and `IBPSA.Fluid.MassExchangers.ConstantEffectiveness`.

**Comment by Mathadon** [comment]
okay, then I'll add it!

**Comment by Mathadon** [comment]
fixed in #920
